### ISSUE TITLE
Helm charts referencing the wrong images

### ISSUE THREAD

**Issue by salaboy** [zeebe-io/zeebe-cluster-helm#107]
`helm install default-zeebe zeebe/zeebe-full-helm --values https://raw.githubusercontent.com/zeebe-io/zeebe-helm-profiles/master/default.yaml`

I Pod was not able to run since it dosn't not find container:
```
Failed to pull image "gcr.io/camunda-researchanddevelopment/zeebe-cluster-helm:0.0.211": rpc error: code = Unknown desc = Error response from daemon: manifest for gcr.io/camunda-researchanddevelopment/zeebe-cluster-helm:0.0.211 not found: manifest unknown: Failed to fetch "0.0.211" from request "/v2/camunda-researchanddevelopment/zeebe-cluster-helm/manifests/0.0.211".
```

**Comment by davidriskified** [comment]
I find it in 211 :-)

**Comment by salaboy** [comment]
For those encountering this issue.. I am working on it, but in the meantime please use version 0.0.131 of the zeebe-full-helm chart.

**Comment by salaboy** [comment]
solved in full 0.0.134 and cluster 0.0.213
